lib/elixir/lib/inspect/algebra.ex

```diff
@@ -565,6 +599,10 @@ defmodule Inspect.Algebra do
     end
   end
 
+  defp ansi(color) do
+    color |> IO.ANSI.format_fragment(true)
```

Comment:
This will always emit the ansi codes. Is there any reason we don't want to check if it's enabled?